gnss/gps_time.py

```diff
@@ -126,19 +134,23 @@ def gpst_to_utc(gpst):
 
     Returns
     -------
-    utc : np.datetime64
+    utc : np.datetime64[ns]
       Returns a np.datetime64 object (or an array of them) that holds
       the UTC representation of the corresponding gpst.
 
     """
-    delta_utc = gps_minus_utc_seconds(gpst)
+    if gpst is dict:
```

Comment:
I am not sure it is True when gpst is a dict. Should it be `isinstance(gpst, dict)` instead?

Reply:
ah you might be right, mypy was complaining about something here so that might have been the issue